scripts-dev/release.py

```diff
@@ -841,5 +845,47 @@ def generate_and_write_changelog(
         os.remove(filename)
 
 
+def build_dependabot_changelog(repo: Repo, current_version: version.Version) -> str:
+    """Summarise dependabot commits between `current_version` and `release_branch`.
+
+    Returns an empty string if there have been no such commits; otherwise outputs a
+    third-level markdown header followed by an unordered list."""
+    last_release_commit = repo.tag("v" + str(current_version)).commit
+    rev_spec = f"{last_release_commit.hexsha}.."
+    commits = list(git.objects.Commit.iter_items(repo, rev_spec))
+    messages = []
+    for commit in reversed(commits):
```

Comment:
Could we pass `author` to `iter_items`?

Reply:
Not that I could see from https://gitpython.readthedocs.io/en/stable/reference.html?highlight=iter_items#git.objects.commit.Commit.iter_items, but maybe it would pass it through to `git rev-list`.